cassandra.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"log"
+
+	"github.com/gocql/gocql"
+)
+
+type cassandraBackend struct {
+	Cluster *gocql.ClusterConfig
+	Session *gocql.Session
+}
+
+func (b *cassandraBackend) createSession() {
+	log.Println("Starting cassandra session")
+	session, err := b.Cluster.CreateSession()
+	if err != nil {
+		log.Fatal("Cassandra init failed")
+	}
+	b.Session = session
+}
+
+func (b *cassandraBackend) EndSession() {
+	log.Println("Ending cassandra session")
+	if b.Session != nil {
+		b.Session.Close()
+		b.Session = nil
+	}
+}
+
+// cassandraBackend must be made with this function
```

Comment:
should config struct be passed here or in the implemented interface methods

Reply:
The dependency is no longer there. Necessary configuration for cassandra is passed during initialization.